### ISSUE TITLE
Only one manifest is emitted when used in multiple configurations

### ISSUE THREAD

**Issue by uhyo** [danethurber/webpack-manifest-plugin#147]
Thank you for the great plugin!

When it is used multiple times in multiple (array) webpack configurations, only one manifest is emitted.

**webpack and webpack-manifest-plugin version**:
- webpack 4.6.0
- webpack-manifest-plugin: 2.0.1

**Reproduction**:
webpack.config.js
```js
const path = require('path');
const ManifestPlugin = require('webpack-manifest-plugin');

const config1 = {
    mode: 'development',
    entry: {
        entry1: './src/entry1.js',
    },
    output: {
        path: path.join(__dirname, './dist1'),
    },
    plugins: [
        new ManifestPlugin(),
    ],
}; 

const config2 = {
    mode: 'development',
    entry: {
        entry2: './src/entry2.js',
    },
    output: {
        path: path.join(__dirname, './dist2'),
    }, 
    plugins: [
        new ManifestPlugin(),
    ],
}; 
   
module.exports = [config1, config2];
```

When we run this we expect `dist1/manifest.json` and `dist2/manifest.json` to be emitted, but actually we get only `dist2/manifest.json`.

A workaround is to use different file names for each configuration, as in:
```js
// in config1
new ManifestPlugin({
  fileName: 'manifest1.json',
})
// in config2
new ManifestPlugin({
  fileName: 'manifest2.json',
})
```

Thanks.

**Comment by mastilver** [comment]
Related #146

**Comment by mastilver** [comment]
I believe it was introduced by: https://github.com/danethurber/webpack-manifest-plugin/pull/140

I didn't have time yet to do any further investigation. Feel free to submit a PR if you have a solution ;)

**Comment by izzyu91** [comment]
I can confirm the bug and that the change from #146 fixes it.

**Comment by uhyo** [comment]
Thanks for reply. I searched for issues but missed the related pull request. Sorry about that.

**Comment by mastilver** [comment]
fixed on `v2.0.2`
